aiohttp_bot/delete_branch.py

```diff
@@ -0,0 +1,14 @@
+import gidgethub.routing
+
+router = gidgethub.routing.Router()
+
+
+@router.register("pull_request", action="closed")
+async def delete_branch(event, gh, *args, **kwargs):
```

Comment:
There's https://github.com/apps/delete-merged-branch/ for this task. So there's no need to duplicate the effort.